tests/e2e/scenarios/test_reborn_webui_v2_legacy_extensions.py

```
@@ -0,0 +1,2361 @@
+        await expect(modal.locator('input[type="text"][placeholder="team-slug"]')).to_be_visible()
+
+        await modal.get_by_role("button", name="Cancel").click()
+        assert harness["setup_submit_requests"] == []
+    finally:
+        await harness["context"].close()
+
+
+async def test_reborn_legacy_configure_modal_setup_url_requires_https(
+    reborn_v2_server, reborn_v2_browser
+):
+    async def open_with_setup_url(setup_url: str):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `open_with_setup_url`

(ANN202)